test/index.test.js

```diff
@@ -553,6 +553,25 @@ describe('config parser', () => {
             }));
     });
 
+    describe('warnAnnotations', () => {
+        it('warning it is not pipeline-level annotation',
+            () => parser(loadData('warn-pipeline-level-annotation.yaml'))
+                .then((data) => {
+                    /* eslint-disable max-len */
+                    assert.match(data.warnAnnotations[0],
+                        /screwdriver.cd\/(chainPR|restrictPR) is not an annotation that is reserved for Pipeline-Level/);
```

Comment:
kind of confused
chainPR/restrictPR are pipeline-level annotations, right?

should the message be more like 
```suggestion
                        /screwdriver.cd\/(chainPR|restrictPR) is an annotation that is reserved for Pipeline-Level/);
```

Reply:
Thanks! I was confused, too. That's right, there is Pipeline-Level. I fixed it.

The code checks that it's not for Job-Level, and it doesn't check that it's for Pipeline-Level. Sometimes it's just a single-letter typo. So it can't be assured that it's for pipeline-Level, it has to be a warning that it's not for Job-Level.